lib/hackerone/client/report.rb

```diff
@@ -75,6 +75,32 @@ def activities
         end
       end
 
+      def program
+        @program || Program.find(relationships[:program][:data][:attributes][:handle])
+      end
+
+      def assign_to_user(name)
+        fail(
```

Comment:
If you still like to have helpful error messages you could generate them if something fails. So just do the operation and if it fails, create a hint-like message saying what the error could me. 

IMO think the API should return what's going on. If we fix the API, we can replace the hinting strategy with the actual errors.